packages/contracts/src/dollar/mocks/MockBondingV1.sol

```diff
@@ -144,12 +144,12 @@ contract Bonding is CollectableDust {
         emit RedeemStreamTimeUpdated(_redeemStreamTime);
     }
 
-    function setBlockRonding(uint256 _blockRonding)
+    function setBlockBonding(uint256 _blockBonding)
```

Comment:
I'm impressed with how many spelling errors there are in this codebase lol cspell is a godsend.

Reply:
indeed. it is quite accurate